laspy/base.py

```diff
@@ -178,14 +215,12 @@ def point_map(self):
     def close(self, flush=True):
         '''Close the data provider and flush changes if _mmap and _pmap exist.'''
         if flush and self.manager.has_point_records:
-            if type(self._mmap) != bool:
-                try:
-                    self._mmap.flush()
-                    self._mmap.close()
-                    self._mmap = False
-                    self._pmap = False
-                except(Exception):
-                    raise laspy.util.LaspyException("Error closing mmap")
+            try:
+                self._mmap.flush()
+                self._mmap.close()
+            except(Exception):
+                self._mmap = False
+                self._pmap = False
```

Comment:
same on the exception catching as above. If we can't flush()/close(), there's nothing we can do about it except zero them out, which is what the method does anyway. Telling the user the map wasn't closed() doesn't help them do anything about it.